jenkins_job_wrecker/modules/triggers.py

```diff
@@ -38,18 +40,211 @@ def reversebuildtrigger(top, parent):
         if child.tag == 'upstreamProjects':
             reverse['jobs'] = child.text
         elif child.tag == 'threshold':
-            pass    # TODO
+            pass  # TODO
         elif child.tag == 'spec':
-            pass    # TODO
+            pass  # TODO
         else:
             raise NotImplementedError('cannot handle reverse trigger '
                                       'setting %s' % child.tag)
 
     parent.append({'reverse': reverse})
 
 
+def process_file_paths(attribute):
+    file_paths = []
+    for file_path_type in attribute:
+        if file_path_type.tag == "com.sonyericsson.hudson.plugins.gerrit.trigger.hudsontrigger.data.FilePath":
+            file_path = {}
+            for file_path_attribute in file_path_type:
+                if file_path_attribute.tag == "compareType":
+                    file_path["compare-type"] = file_path_attribute.text
+                elif file_path_attribute.tag == "pattern":
+                    file_path["pattern"] = file_path_attribute.text
+            file_paths.append(file_path)
+        else:
+            raise NotImplementedError("Not implemented file path type: ", file_path_type.tag)
+    return file_paths
+
+
 def gerrittrigger(top, parent):
-    pass
+    gerrit_trigger = {}
+    for child in top:
+        print(child.tag)
+        if child.tag == "gerritProjects":
```

Comment:
I'd extract this part into a separate function for readability, also the same for "triggerOnEvents"